### PR TITLE
Make square frame for resignImageView

### PR DESCRIPTION
**Pull request by patrickxb**
cc @keybase/react-hackers 

I think this solution actually works for rotating iPad.

### CODE REVIEW

File: shared/ios/Keybase/AppDelegate.m [hunk 1]
```diff
@@ -85,7 +85,17 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   [self.window makeKeyAndVisible];
 
   // To simplify the cover animation raciness
-  self.resignImageView = [[UIImageView alloc] initWithFrame:self.window.bounds];
+  
+  // make a square of the biggest screen dimension to handle rotate snapshot
```
Comment (chrisnojima) [inline shared/ios/Keybase/AppDelegate.m:89]: i think this isn't actually what you want to do. i think you can just center the subview, something like
`self.resignImageView.center = [self.window convertPoint:self.window.center fromView:self.window.superview];`

Comment (patrickxb) [reply shared/ios/Keybase/AppDelegate.m:89]: I tried this, it doesn't work.  Same problem.

**Comment by patrickxb** [comment]
Verified on iPad pro 12.9" device, this fix works.

File: shared/ios/Keybase/AppDelegate.m (hunk 1 above)
Comment (chrisnojima) [reply shared/ios/Keybase/AppDelegate.m]: its likely cause its resizing itself based on the parent etc, anyways i don't htink this is a great fix but whatever

**Pull request merged**